src/go/MONIT/annotationManager.go

```diff
@@ -40,6 +40,9 @@ var annotation string
 // annotation ID for deleting single annotation
 var annotationID int
 
+// limit for fetching number of annotations at a time
+var grafanaLimit string
```

Comment:
the limit is an integer data-type, please do not mix data types when it is not necessary

Reply:
fixed.